### DISCUSSION TITLE
Make author sidebar non-sticky

### DISCUSSION THREAD

**Discussion by Llannelongue** [mmistakes/minimal-mistakes#2612] [Q&A] [answered]
I'd like to make the left author sidebar non sticky. I've tried using a custom sidebar (without the sticky class) in `_include` using:
```
<div class="sidebar">
   ...
</div>
``` 

However, for some reason, this is automatically replaced by `class="sidebar sticky"` when building the site. A dirty fix is to use:
```
<div class="sidebar" style="position:static">
   ...
</div>
``` 

Any tips on the best way to do it?

Thanks!

**Comment by mmistakes** [comment, marked as answer]
Hmm not sure on it being replaced with, but all that's needed is to remove the `sticky` class from [this `_include`](https://github.com/mmistakes/minimal-mistakes/blob/master/_includes/sidebar.html#L2).
